mod/crepair.php

```diff
@@ -78,18 +82,18 @@ function crepair_post(App $a) {
 		local_user()
 	);
 
-	if($photo) {
+	if ($photo) {
 		logger('mod-crepair: updating photo from ' . $photo);
 		require_once("include/Photo.php");
 
 		update_contact_avatar($photo,local_user(),$contact['id']);
 	}
 
-	if($r)
+	if ($r) {
```

Comment:
That came from an UPDATE statement and I have not clearly checked it that I can really use `dbm::is_result()` here.